.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Looks like all the eslint files got committed to this PR; we either need to merge #23 before this one in order to keep things separate, or else we should rebuild this PR so that it only includes the code changes and not the tools.

Reply:
These are faithful to #23 so they'll be good after #23 is merged or I can compare this to the eslint branch.

Reply:
Sounds good. As long as no further tool configuration changes are needed, the easiest thing then would be to merge #23 first, then merge dev into this branch, and then as long as there are no conflicts, it should all be clean.